.github/workflows/check-ndi-sdk.yaml

```diff
@@ -0,0 +1,30 @@
+name: Check for updates
+on:
+  schedule:
+  - cron: "* * * * 1"
+  workflow_dispatch: 
+
+jobs:
+  flatpak-external-data-checker:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        branch: [ master ]
+    
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ matrix.branch }}
+
+      - uses: docker://ghcr.io/flathub/flatpak-external-data-checker:latest
+        env:
+          GIT_AUTHOR_NAME: Flatpak External Data Checker
+          GIT_COMMITTER_NAME: Flatpak External Data Checker
+          GIT_AUTHOR_EMAIL: 41898282+github-actions[bot]@users.noreply.github.com
+          GIT_COMMITTER_EMAIL: 41898282+github-actions[bot]@users.noreply.github.com
+          EMAIL: 41898282+github-actions[bot]@users.noreply.github.com
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          args: --filter-type  ${{ matrix.filter-type }} --update --never-fork com.obsproject.Studio.Plugin.DistroAV.json
```

Comment:
matrix.filter-type doesn't seem to exist

Reply:
Need to set it, I forgot to do it on the NDI repo… which explain things.